Applications/Games/Mass Effect/Steam/script.js

```diff
@@ -8,8 +8,7 @@ new SteamScript()
     .postInstall((wine, wizard) => {
         wizard.message(
             tr(
-                "If you have sound issues, please edit the BIOEngine.ini and/or BaseEngine.ini file in {0}/drive_c/Program Files/Steam/steamapps/common/Mass Effect/Engine/Config/\n\nAnd add the following under [ISACTAudio.ISACTAudioDevice] :\n\nDeviceName=Generic Software\nUseEffectsProcessing=False\n\n",
-                wine.prefixDirectory
+                "If you have sound issues, please edit the BIOEngine.ini and/or BaseEngine.ini file in {0}/drive_c/Program Files/Steam/steamapps/common/Mass Effect/Engine/Config/\n\nAnd add the following under [ISACTAudio.ISACTAudioDevice] :\n\nDeviceName=Generic Software\nUseEffectsProcessing=False\n\n"
```

Comment:
This doesn't look correct. The "{0}" is supposed to be replaced by the Wine prefix directory. Now it will probably just print "{0}".
Untested:
```suggestion
                "If you have sound issues, please edit the BIOEngine.ini and/or BaseEngine.ini file in {0}/drive_c/Program Files/Steam/steamapps/common/Mass Effect/Engine/Config/\n\nAnd add the following under [ISACTAudio.ISACTAudioDevice] :\n\nDeviceName=Generic Software\nUseEffectsProcessing=False\n\n", wine.prefixDirectory()
```

Reply:
Then why #1173 is an issue?

Reply:
The `()` are missing.

Reply:
Fixed